cmd/metrics-v3-cache.go

```diff
@@ -24,6 +24,10 @@ import (
 	"github.com/minio/minio/internal/cachevalue"
 )
 
+// initialDriveIOStats is used to calculate "per second"
+// values for IOStat related disk metrics e.g. reads/sec.
+var initialDriveIOStats *diskIOStats
```

Comment:
Is it possible to re-use the existing functions and types? @anjalshireesh 

`madmin.DiskIOStats` for example?

Reply:
`metrics-resource.go` implements its own caching mechanism, whereas `metrics-v3` has introduced a standard caching mechanism across all metrics. Because of this most of the existing functions cannot be reused as is. Having said that, please do point out if you find any specific missed reuse opportunities.